pages/common/az-config.md

```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Manage Azure CLI configuration.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Get all configurations:
```

Comment:
Maybe this will fit our generic wording better (Please apply for the command below as well)
```suggestion
- Print all configurations:
```

Reply:
Yes, agree. Commiting the change!